mooncake-common/common.cmake

```diff
@@ -83,3 +83,9 @@ if (WITH_METRICS)
   add_compile_definitions(WITH_METRICS)
   message(STATUS "metrics is enabled")
 endif()
+
+
+set(GFLAGS_USE_TARGET_NAMESPACE "true")
+find_package(gflags REQUIRED)
+find_package(glog REQUIRED)
+find_package(yalantinglibs CONFIG REQUIRED)
```

Comment:
A question: Can installing yalantinglibs using either yum/apt or make install enable this cmake config?